lib/connect.js

```diff
@@ -210,7 +210,17 @@ const getTasks = (model, setupListeners) => {
       validOptions.useNewUrlParser = true;
       validOptions.useUnifiedTopology = true;
 
-      const mongoClient = new MongoClient(model.driverUrl, validOptions);
+      const modelClone = clone(model, true);
```

Comment:
Instead of `clone` here I think we want to use `cloneDeep` to ensure we clone the driver options https://lodash.com/docs/4.17.15#cloneDeep